src/nodejs/src/font-size-init.ts

```diff
@@ -0,0 +1,100 @@
+import { Log } from 'logging';
+import { DeviceInfo } from 'device-info';
+import { ScreenSize } from '../../dotnet/UI.Blazor/Services/ScreenSize/screen-size';
+
+const { debugLog } = Log.get('FontSizeInit');
+
+const FontSize : {[title: string]: string} = {
+    'Small': '14px',
+    'Medium': '16px',
+    'Large': '18px',
+    'XL': '20px',
+    'XXL': '24px',
+}
+
+export class FontSizeInit {
+    private static fontSize= FontSize['Medium'];
+
+    public static init(): void {
+        debugLog?.log(`init`);
+
+        const root = document.querySelector(':root');
+        const rs = window.getComputedStyle(root);
+
+        const rootFontSize = rs.getPropertyValue('--font-size');
+        const localFontSize = localStorage.getItem('font-size');
+
+        if (localFontSize == null) {
+            this.isLocalFontSizeNull();
+        } else {
+            if (rootFontSize != localFontSize) {
+                if (!this.checkLocalFontSizeValues(localFontSize)) {
+                    // get default
+                    this.isLocalFontSizeNull();
+                } else {
+                    this.fontSize = localFontSize;
+                }
+            }
+        }
+
+        if (localFontSize != this.fontSize) {
+            localStorage.setItem('font-size', this.fontSize);
+        }
+        if (rootFontSize != this.fontSize) {
+            (root as HTMLElement).style.setProperty('--font-size', this.fontSize);
+        }
+
+        ScreenSize.change$.subscribe(_ => this.init());
```

Comment:
There is an issue here, that every screen size change will add one more subscription on screen size change.
You can set a flag that you subscribed already and next time skip subscription.
Or refactor code to something like:
public static init(): void {
    this.initInternal();
    ScreenSize.change$.subscribe(_ => this.initInternal());
}